homeassistant/components/synology_dsm/binary_sensor.py

```diff
@@ -18,15 +20,19 @@
 
 
 async def async_setup_entry(
-    hass: HomeAssistant, entry: ConfigEntry, async_add_entities
+    hass: HomeAssistant, entry: ConfigEntry, async_add_entities: Callable
```

Comment:
Please use new type here 🙂
```suggestion
    hass: HomeAssistant, entry: ConfigEntry, async_add_entities: AddEntitiesCallback
```